nio.json

```diff
@@ -0,0 +1,162 @@
+      "conf": "etc/logging.json"
+    },
+    "modules": {
+      "communication": {
+        "pubkeeper": {
+          "client": {
+            "token": "",
+            "host": "",
+            "port": "",
+            "ca_chain": "",
+            "secure": "",
+            "validate": true,
+            "websocket_ping_interval": 10,
+            "brews": [
+              "pubkeeper.brew.local.brew.LocalBrew",
+              "pubkeeper.brew.websocket.brew.WebsocketBrew"
+            ]
+          },
+          "server": {
+            "standalone": false,
+            "jwt_secret": "",
+            "ip_address": "",
+            "interface": "",
+            "port": "",
+            "connect_timeout": 2,
+            "websocket_ping_interval": 10,
+            "use_localhost": false
+          },
+          "websocket": {
+            "ws_host": "",
+            "ws_port": "",
+            "ws_secure": ""
+          },
+          "pubkeeper.brew.zmq.brew.ZMQBrew": {
+            "ip_address": "",
+            "publisher_min_port": 9010,
+            "publisher_max_port": 9199,
+            "linger": 1000,
+            "hwm_size": 100000,
+            "max_sockets": 1023,
+            "max_io_thread_count": 1,
+            "encryption": {
+              "allowed_ip_addresses": "",
+              "publisher_public_keys_dir": "",
+              "publisher_server_public_cert": "",
+              "publisher_server_secret_cert": "",
+              "subscriber_client_secret_cert": ""
+            }
+          }
+        }
+      },
+      "scheduler": {
+        "custom": {
+          "min_interval": 0.1,
+          "resolution": 0.1
+        }
+      },
+      "security": {
+        "basic": {
+          "users": "etc/users.json",
+          "permissions": "etc/permissions.json"
+        },
+        "oauth2": {
+          "permissions": "etc/permissions.json",
+          "client_id": "407408718192.apps.googleusercontent.com",
+          "validate_url": "https=/www.googleapis.com/oauth2/v2/tokeninfo?access_token=%s"
+        },
+        "jwt": {
+        }
+      },
+      "web": {
+        "cherrypy": {
+          "server_thread_pool": 50,
+          "request_show_tracebacks": false,
+          "cors_allow_origin": "*"
+        }
+      }
+    },
+    "components": {
+      "snmp": {
+        "host": "127.0.0.1",
+        "port": 161,
+        "community_index": "agent",
+        "community_name": "public",
+        "trap_host": "127.0.0.1",
+        "trap_port": 162,
+        "trap_community_index": "agent",
+        "status_traps": "error"
+      },
+      "management_publisher": {
+        "topic": "management",
+        "add_instance_id": true
+      },
+      "rest": {
+        "port": "",
+        "host": "",
+        "network_retry_interval": 30,
+        "ssl_certificate": "",
+        "ssl_private_key": "",
+        "ssl_certificate_chain": ""
+      },
+      "diagnostic": {
+        "api_url_prefix": "https://api.nio.works/v1/diagnostic",
```

Comment:
This is an environment variable, kindof.  Depending on if the instance is running on stage or not it will use `api.` or `stage-api.`